### ISSUE TITLE
Provide the ability within the Azure Portal (via Arc) to link AKS HCI clusters to the hosting Azure Stack HCI cluster

### ISSUE THREAD

**Issue by dmc-tech** [Azure/aks-hci#56]
**Title**: Provide the ability within the Azure Portal (via Arc) to link AKS HCI clusters to the hosting Azure Stack HCI cluster

**Description**:
Currently, there is no relationship between the Azure Stack HCI cluster  and each of the AKS HCI K8s clusters that are running on there in the Azure Portal (via Arc).  
It would be a really useful feature to show the relationships; e.g. if there is a performance issue on the HCI cluster, it could affect the workloads running on AKS HCI.  It will help with service management discovery too.

**Comment by comnam90** [comment]
Yeah this is a must. I'd love to eventually see AKS as like an extension in the Azure Portal for Azure Stack HCI Clusters so it can be deployed through Azure to clusters
